src/aspire/source/image.py

```diff
@@ -0,0 +1,749 @@
+import logging
+import os.path
+
+import mrcfile
+import numpy as np
+import pandas as pd
+from scipy.spatial.transform import Rotation as R
+
+from aspire.image import Image, normalize_bg
+from aspire.image.xform import (
+    Downsample,
+    FilterXform,
+    IndexedXform,
+    LambdaXform,
+    Multiply,
+    Pipeline,
+)
+from aspire.operators import (
+    IdentityFilter,
+    LambdaFilter,
+    MultiplicativeFilter,
+    PowerFilter,
+)
+from aspire.storage import MrcStats, StarFile, StarFileBlock
+from aspire.utils import ensure
+from aspire.utils.coor_trans import grid_2d
+
+logger = logging.getLogger(__name__)
+
+
+class ImageSource:
```

Comment:
Why the move? (I don't object. Just curious.)

Reply:
I started by trying to fix on the simulation import level towards our target hierarchy (wanted from aspire.source import Simulation). This required adding Simulation to `source/__init__.py`. 

However, Simulation imports ImageSource. So in original code, I'd need to bury the import near the bottom of the __init__ after defining ImageSource. then special case why I did that to the linters (this is generally a taboo thing to do which they would complain about).

_or_  just break out the classes into their own file and make the `__init__` naive, now only performing imports for that package.

Reply:
Great. I prefer this style anyhow.